src/coreclr/src/vm/methodtable.cpp

```diff
@@ -2575,7 +2575,8 @@ bool MethodTable::ClassifyEightBytesWithManagedLayout(SystemVStructRegisterPassi
 bool MethodTable::ClassifyEightBytesWithNativeLayout(SystemVStructRegisterPassingHelperPtr helperPtr,
                                                     unsigned int nestingLevel,
                                                     unsigned int startOffsetOfStruct,
-                                                    bool useNativeLayout)
+                                                    bool useNativeLayout,
```

Comment:
Do we need the `useNativeLayout` argument? The first thing that this method does is to assert that it is true.